sound/soc/sof/ipc4-topology.c

```diff
@@ -2413,9 +2415,14 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 
 		msg->extension &= ~SOF_IPC4_MOD_EXT_PPL_ID_MASK;
 		msg->extension |= SOF_IPC4_MOD_EXT_PPL_ID(pipe_widget->instance_id);
+
+		dev_dbg(sdev->dev, "Create widget %s ID %#x instance %d - pipe %d - core %d\n",
+			swidget->widget->name, module_id, swidget->instance_id,
+			swidget->pipeline_id, swidget->core);
+	}else {
+		dev_dbg(sdev->dev, "Create widget %s instance %d - pipe %d - core %d\n",
+			swidget->widget->name, swidget->instance_id, swidget->pipeline_id, swidget->core);
```

Comment:
Would not be beneficial to print this differently:
if ()
`Create widget for module: %s, ID: %d, instance: %d, pipeline: %d, core: %u`
else
`Create widget for pipeline: %s, instance: %d, pipeline: %d, core: %u`